packages/nep141-erc20/src/utils/index.js

```diff
@@ -0,0 +1,85 @@
+import Web3 from 'web3'
+import { getEthProvider } from '@near-eth/client/dist/utils'
+
+
+/**
+ * Search for a dropped and replaced transaction (speed up or cancel)
+ * @param {Number} safeHeightBeforeEthTx Lower search bound
+ * @param {*} txInfo Transaction information to find the replacement
+ * @param {*} eventVerifier Check event information to verify if it was a speed up
+ */
+export async function handleEthTxReplaced(
+  safeHeightBeforeEthTx,
+  {broadcastedNonce, txFrom, txTo},
+  {eventName, contractAbiText, validateEvent}
+) {
+  const web3 = new Web3(getEthProvider())
+
+  const currentNonce = await web3.eth.getTransactionCount(txFrom, 'latest')
```

Comment:
[getTransactionCount](https://web3js.readthedocs.io/en/v1.3.4/web3-eth.html?highlight=gettransactioncount#gettransactioncount) returns "The number of transactions sent from the given address", not a nonce. Does comparing this to `pendingWhateverTx.nonce`, which will be "The number of transactions made by the sender prior to this one", make sense? 

From the docs, `getTransactionCount` seems like it might return the number of _pending_ transactions, so I just want to check my understanding here. Seems like you know what you're doing 😄

Reply:
Lets take a simple example: If `1` transaction was made (getTransactionCount = 1), then that transaction had `nonce = 0`. So the currentNonce to be used in the next transaction is `1`

`'latest'` means the head of the chain (last block mined)